### DISCUSSION TITLE
[Art competition] Widening

### DISCUSSION THREAD

**Discussion by Superstellar** [prosyslab-classroom/cs524-program-analysis#278] [Art competition]
![DALL·E 2022-11-30 14 21 36 - A rocket flying next to the stairway to heaven, an impressionism painting](https://user-images.githubusercontent.com/38059780/204714446-6365e687-20ff-45fb-8532-77a23b3563b5.png)

## Short Description
For me, widening means "A rocket flying next to the stairway to heaven, an impressionism painting".

## Long Description (up to 3 sentences)
During the iterative computation to reach the fixed point, we may have to walk through infinite amount of steps. However, we can reach to the heaven of the fixed point by taking a rocket named widening.

**Comment by KihongHeo** [comment]
Then, what is your least fixed point?

**Comment by Superstellar** [comment]
I wanted to express least fixed point as clouds because there will be more spaces above the cloud, but i failed to express it through Dall-E.
